packages/autorest.typescript/test/commands/smoke-test-list.ts

```diff
@@ -230,6 +230,19 @@ export const readmes: SpecDefinition[] = [
       AutorestParams.GenerateSamples
     ],
     buildTag: "ci_rlc"
+  },
+  {
+    path: joinPath(
+      `${__dirname}`,
+      "..",
+      "./smoke/swagger/anomaly-detector-rest.md"
```

Comment:
why add this one into smoke-test list ?

Reply:
I want to enable more sample generation testing for rlc